### PR TITLE
Allowlists: mark several entries as wontfix, add note to inspect.Parameter entries

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: tests/stubtest_allowlists/py3_common.txt [hunk 1]
```diff
@@ -392,6 +393,13 @@ email.contentmanager.set_bytes_content
 email.contentmanager.set_message_content
 email.contentmanager.set_text_content
 email.iterators.walk
+pipes.Template.makepipeline
```
Comment (Akuli) [inline tests/stubtest_allowlists/py3_common.txt:396]: Why was this moved?

File: tests/stubtest_allowlists/py3_common.txt [hunk 2]
```diff
@@ -392,6 +393,13 @@ email.contentmanager.set_bytes_content
...
+pipes.Template.open_r
+pipes.Template.open_w
+sunau.Au_read.initfp
+sunau.Au_write.initfp
+wave.Wave_read.initfp
+wave.Wave_write.initfp
```
Comment (Akuli) [inline tests/stubtest_allowlists/py3_common.txt:402]: These seem to be implementation details that just happen to not be named with `_` in front. It might be good to add a comment that mentions this.

Comment (AlexWaygood) [reply tests/stubtest_allowlists/py3_common.txt:402]: This whole section of the allowlist is subtitled "undocumented implementation details"

File: tests/stubtest_allowlists/py3_common.txt (hunk 1 above)
Comment (AlexWaygood) [reply tests/stubtest_allowlists/py3_common.txt:396]: Undocumented and there's no docstring, seems like it belongs in the "undocumented implementation details" section?

Comment (AlexWaygood) [reply tests/stubtest_allowlists/py3_common.txt:396]: Happy to add it to the stub if you think it's useful

Comment (AlexWaygood) [reply tests/stubtest_allowlists/py3_common.txt:396]: Seems to me like it's just a helper function used by `open_r`, `open_w` and `copy`

**Review by Akuli — APPROVED** [review]
